src/main/java/fi/hsl/transitdata/cancellation/util/TimeUtils.java

```diff
@@ -31,4 +42,81 @@ public static Optional<Long> toUtcEpochMs(String localTimestamp, String zoneId)
             return Optional.empty();
         }
     }
+    
+    /**
+     * Returns dates within the given time period. For example, if time period is from 2024-01-02 to 2024-01-05, this
+     * method returns the following dates as strings: '20240102', '20240103', '20240104' and '20240105'
+     * @param validFrom alert valid from timestamp
+     * @param validTo alert valid to timestamp
+     * @return list of dates as string, each date has format 'YYYYMMDD'
+     * @exception throws RuntimeException if parameter validFrom and/or validTo is null, or if validFrom is after validTo
+     */
+    public static List<String> getDates(LocalDateTime validFrom, LocalDateTime validTo) {
```

Comment:
I'm not sure if List<String> is good data structure for pair. Normally it wouldn't make sense but I haven't evaluated context so it might be required here.

Either way, check and consider changes.

Reply:
I misinterpreted what the method was doing. This is non-issue.